application/classes/Logger.php

```diff
@@ -9,41 +9,46 @@ public static function instance() {
 	}
 	
 	private static function loadLogger() {
-		$prov_config = Kohana::$config->load('logger');
+		if (is_null(Kohana::$config)) {
+			$prov_config = [ 'type' => 'syslog', 'config' => 'controll' ];
+		} else {
+			$prov_config = Kohana::$config->load('logger');
+		}
 		$fullclass = "Log_" . str_replace('/', '_', ucfirst($prov_config['type']));
-		$logger = new $fullclass($prov_config);
-		Kohana::$log->attach($logger);
+		$logger = new $fullclass(array_key_exists('config', $prov_config) ? $prov_config['config'] : $prov_config);
+		if (!is_null(Kohana::$log))
```

Comment:
Hmm.. this code shouldn't have been seen here - its older and not part of the feature. the github pull requests don't play well with the git flow we normally use. Anyway, this is a mess that was meant to handle an edge case where a crash during an early initialization. I'll see about fixing this up.